src/styles/cylc/_tree.scss

```diff
@@ -15,7 +15,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-@import '../../../node_modules/vuetify/src/styles/styles';
+@import '~vuetify/src/styles/styles';
```

Comment:
The `~` is the correct way. The other syntax is valid depending on your build tools. And I think `~@` means that it's a file in the current project, while `~` means it's a file in the `node_modules` folder. This broke with Yarn V3 with PnP mode (plug-and-play, when you have no `node_modules` folder, but instead each dependency loaded from a cache, from what I understand).